store/plugins/websocket.ts

```diff
@@ -36,7 +39,31 @@ export const websocketPlugin: Plugin<AppState> = (store) => {
       keepAliveInterval = window.setInterval(() => {
         sendWebsocketMessage('KeepAlive');
       }, state.socket.message.Data * 1000 * 0.5);
-    } else if (mutation.type === 'SOCKET_ONCLOSE') {
+    }
+    // Update items when changed
+    else if (
+      mutation?.type === 'SOCKET_ONMESSAGE' &&
+      state.socket.message.MessageType === 'LibraryChanged'
```

Comment:
There isn't an enum we can use with all the websocket message types?

Reply:
There is one in the server, but it's not exposed to the API at the moment.

Reply:
We should raise an issue for this in server 👍🏻.